### ISSUE TITLE
Possible to skip over captcha badge links when tabbing?

### ISSUE THREAD

**Issue by jsunsawyer** [verbb/formie#519]
Regardless of its placement, captcha badge links are being tabbed to after the last form field – just before the submit button.

Also, the captcha itself tabs through 4 links (privacy and terms twice/redundantly), making it even worse when trying to submit a form.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.9](https://github.com/verbb/formie/releases/tag/1.4.9)
